app/templates/main/userProfile.html

```diff
@@ -469,20 +469,29 @@ <h5 class="modal-title">Add Notes</h5>
               {% if vis != "" %}
                 <p><b>This note will only be visible to you, CELTS Student Staff, and Administrators.</b></p>
               {% endif %}
-              <label class="form-label {{vis}}" for='inputEventTerm'><strong>Visibility</strong></label>
+
+              <label class="form-label d-none" for='inputEventTerm' id="noteDrop"><strong>Bonner notes are visible to the students and admins</strong></label>
```

Comment:
This message should be more like "This note will only be visible to Ebenezer and CELTS administrators."